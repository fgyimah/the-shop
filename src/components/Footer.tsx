import React from 'react';
import styled from 'styled-components';

const Footer: React.FC = () => {
	return (
		<StyledDiv>
			<p>Nothing but the best in products and customer service</p>
			<p>
				Â© {new Date().getFullYear()} <strong className="logo">THEShop, </strong> all rights reserved
			</p>
		</StyledDiv>
	);
};

const StyledDiv = styled.div`
	background-color: #eee;
	align-items: center;
	justify-content: space-between;
	display: flex;
	height: 100px;
	padding: 1rem;

	.logo {
		font-weight: 700;
		color: #000;
	}
`;

export default Footer;
