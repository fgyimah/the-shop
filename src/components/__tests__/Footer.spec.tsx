import { render } from '@testing-library/react';
import Footer from '../Footer';

describe('Component Test - Footer', () => {
    it('matches snapshot', () => {
        const { asFragment } = render(<Footer />);
        expect(asFragment()).toMatchSnapshot();
    });
});
