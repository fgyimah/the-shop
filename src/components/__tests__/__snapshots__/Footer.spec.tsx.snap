// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component Test - Footer matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="sc-bdfBwQ kkhNPp"
  >
    <p>
      Nothing but the best in products and customer service
    </p>
    <p>
      Â© 2021 
      <strong
        class="logo"
      >
        THEShop, 
      </strong>
       all rights reserved
    </p>
  </div>
</DocumentFragment>
`;
